src/pages/NotFound/index.tsx

```diff
@@ -1 +1,29 @@
-export const NotFound = () => <p>404 - page not found</p>;
+import { Box, Button, Stack, ThemeProvider, Typography } from '@mui/material';
+import { theme } from '../../themes';
+
+
+export const NotFound = () => 
+    <Box
+    sx={{
+    width: '100%',
+    minHeight: '100vh',
+    display:'flex',
+    alignItems:'center',
+    justifyContent:'center'
+    }}
+    >
+      <Stack
+      direction="column"
+      justifyContent="center"
+      alignItems="center"
+      spacing={2}
+      >
+        <ThemeProvider theme={theme}>
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSQAw3md2--EMzzKYG7AyImi9BeJvaJLO4riQ&usqp=CAU" alt="telescope" style={{borderRadius:'50%'}}/>

```

Comment:
save image in assets folder inside component and use it
get rid of inline styles. make it in styles.ts with styled component